pipeline_dp/aggregate_params.py

```diff
@@ -0,0 +1,41 @@
+"""AggregateParameters containing parameters to pass to the DPEngine."""
```

Comment:
maybe something like

Contains classes used for specifying DP aggregations parameters.